### ISSUE TITLE
Is it possible to segregate the final output into each seperate label and have its original colour from the input image?

### ISSUE THREAD

**Issue by mdrameez1412** [levindabhi/cloth-segmentation#16]

**Comment by yaomilwaukee** [comment]
hello
the link of pre-trained model download doesn't work, Did you download the pre-trained model file? if did, could you share it ?

**Comment by wildoctopus** [comment]
Hi @mdrameez1412 

For pretrained model download with inference script for Alpha Image generation and Cloth segmentation, please check this 
https://github.com/wildoctopus/huggingface-cloth-segmentation

Have cleaned, modified the scripts to run on latest pytorch and other libraries and Its very simple to use as well. 
The script will automatically download the pretrained model. So no worry at all. 
 
You guys can also check the live demo at huggingface - https://huggingface.co/spaces/wildoctopus/cloth-segmentation

If you guys like this, please drop a star to my repo.

Thanks :)
